### ISSUE TITLE
componentWillReceiveProps has been renamed,

### ISSUE THREAD

**Issue by qipoy** [moroshko/react-autosuggest#675]
**everything it's work,  but I got a warning**, 

```
react-dom.development.js:11495 Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: Autosuggest, Autowhatever
```

**Comment by rlueder** [comment]
Here's a PR that should take care of it: https://github.com/moroshko/react-autosuggest/pull/681

**Comment by aberezkin** [comment]
Fixed in 10.0.0
